flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -302,6 +302,15 @@
 	 */
 	public static final String JOB_MANAGER_WEB_LOG_PATH_KEY = "jobmanager.web.log.path";
 
+	/**
+	 * Config parameter indicating whether jobs can be uploaded and run from the web-frontend.
+	 */
+	public static final String JOB_MANAGER_WEB_SUBMISSION_KEY = "jobmanager.web.submit.allow";
```

Comment:
Think of a config as a JSON object. There is an object "jobmanager.web" and it contains an object "submit" with a field "allow."

1) I think "enable" is better than "allow".
2) You have two different nested objects, called "submit" and "upload". Since they refer to the same function, makes sense to fuse them, in my opinion.